### ISSUE TITLE
Push step is not respecting the domain setting

### ISSUE THREAD

**Issue by mozts2005** [Microsoft/vsts-cloudfoundry#22]
```TS
// Extension\CloudFoundry\cloudfoundry.ts
 if (tl.getBoolInput('useRoute')) {
                let domainOption: string = tl.getInput('domainOptions');
                if (domainOption === 'custom') {
                    if (tl.getInput('domain')) {
                        cfPush.arg('-d');
                        cfPush.arg(tl.getInput('domain'));
                    }
                }

                if (tl.getInput('host')) {
                    cfPush.arg('--hostname');
                    cfPush.arg(tl.getInput('host'));
                }
            } else {
                cfPush.arg('--no-route');
            }
```

There is no option for **"domainOptions"** so it is defaulted to empty srtring. check should be removed

**Comment by mozts2005** [comment]
fixed in PR #19

**Comment by madhurig** [comment]
Thanks for your contribution. A new version (1.147.0) of the extension with this fix has been published. The changes will be automatically deployed to your Azure DevOps organization. Let us know if you see any issues.
